server.js

```diff
@@ -402,6 +449,27 @@ var pingStorage = {
   removeItemSync: storage.removeItemSync
 }
 
+storage.initSync({
```

Comment:
What the heck, I _swear_ I left a comment about using https://github.com/simonlast/node-persist#createoptions---synchronous-static-method but there's no record of it on this page.